cdap-docs/developer-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -687,13 +687,44 @@ CDAP pipelines as of version |version|.
             }
           }
 
+   * - ``toggle-switch``
+     - - ``on``: 'On' state of the switch, defined by
+         - ``value``: value to be set for the plugin property
+         - ``label``: a label to be rendered in UI for the 'On' state, limited to 64 characters.
+       - ``off``: 'Off' state of the switch, defined by
+         - ``value``: value to be set for the plugin property
+         - ``label``: a label to be rendered in UI for the 'Off' state, limited to 64 characters.
+       - ``default``: default value for the widget
+     - ``string``
+     - A toggle switch widget that allows toggling between 'on' and 'off' states, and supports setting a custom value
```

Comment:
I think the second part of this statement is unnecessary. Let's just stick to "A toggle switch widget that allows toggling between 'on' and 'off' states". If we're talking about custom values here, why not custom labels. I think that level of detail is in the attributes description, so no need to repeat it here.